src/App.css

```diff
@@ -55,6 +55,7 @@ h3 {
   width: 50px;
   border-radius: 100%;
   border: 2px solid rgba(0, 0, 0, 0.35);
+  margin-left: 25px;
 }
 
```

Comment:
- [ ] Great job! However, I suggest paying attention to even details in order to stick to the design. For example, there are differences between the Zeplin and your design in `spacing and colors`. Take a look at the below screenshot to see 👇 
- Zeplin design 👇 
![Screenshot (202)](https://user-images.githubusercontent.com/67087664/155549666-12e1b455-661a-474c-9021-6ca32346ca76.png)
- Your design 👇 
![Screenshot (203)_LI](https://user-images.githubusercontent.com/67087664/155549759-441aadcf-a9a7-4a3b-8934-a262d56b7d2e.jpg)
Kindly work on the differences and try to stick to the given design 🙏

Reply:
Thank you. I will work on it . 👍